src/core/Metadata.h

```diff
@@ -52,6 +52,7 @@ class Metadata : public ModifiableObject
         bool recycleBinEnabled;
         int historyMaxItems;
         int historyMaxSize;
+        int autosaveDelay;
```

Comment:
The best practice to name time based variables is using a specific unit. In your case I would name as `autosaveDelaySec`.

Reply:
Fixed in [3a1e023](https://github.com/keepassxreboot/keepassxc/pull/9100/commits/3a1e0231d88af17d161555fa7182d6b9166b162a)